docs/src/configuration/services/_index.md

```diff
@@ -1,177 +1,182 @@
+An example relationship to connect to the databases given in the [example in step 1](#1-configure-the-service):
 
-## Service timezones
+```yaml
+relationships:
+    mysql_database: "database1:mysql"
+    postgresql_database: "database2:postgresql"
+```
 
-All services have their system timezone set to UTC by default.  In most cases that is the best option.  For some applications it's possible to change the application timezone, which will affect only the running application itself.
+As with the service name, you can give the relationship any name you want
+with lowercase alphanumeric characters, hyphens, and underscores.
+It helps if the service name and relationship name are different, but it isn't required.
 
-* MySQL - You can change the per-connection timezone by running SQL `SET time_zone = <timezone>;`.
-* PostgreSQL - You can change the timezone of current session by running SQL `SET TIME ZONE <timezone>;`.
+Each service offers one or more endpoints for connections, depending on the service.
+An endpoint is a named set of credentials to give access to other apps and services in your project.
+If you don't specify one in the [service configuration](#service-options), a default endpoint is created.
+The default varies by service, generally being its type (such as `mysql` or `solr`).
 
-## Using the services
+## Available services
 
-In order for a service to be available to an application in your project (Platform.sh supports not only multiple backends but also multiple applications in each project) you will need to refer to it in the [.platform.app.yaml](/configuration/app/_index.md) file which configures the *relationships* between applications and services.
+The following table presents the available service types and their versions.
+Add them to the `type` key of the [service configuration](#1-configure-the-service) in the format `type:version`.
 
-## Endpoints
+<!-- To update the versions in this table, use docs/data/registry.json -->
+{{< readFile file="src/registry/images/tables/services_supported.md" markdownify="true">}}
 
-All services offer one or more `endpoints`.  An endpoint is simply a named set of credentials that can be used to give access to other applications and services in your project to that service.  Only some services support multiple user-defined endpoints.  If you do not specify one then one will be created with a standard defined name, generally the name of the service type (e.g., `mysql` or `solr`).  An application container, defined by a `.platform.app.yaml` file, always exposes an endpoint named `http` to allow the [router](/configuration/routes/_index.md) to forward requests to it.
```

Comment:
Do we need to keep these details? 
>An application container, defined by a `.platform.app.yaml` file, always exposes an endpoint named `http` to allow the [router](/configuration/routes/_index.md) to forward requests to it.

We do show the router in site architecture graphs, and it's more a service container than an app container, from my PoV at least 🤔

Reply:
I would think the details about the router and the app container endpoint would fall under the routing description. Mostly, I think it's not actionable on the user's end. It's a description of what is happening in the background, but I don't think it affects anything the user does. Or I didn't see where it would. If it does, I'm happy to add the details about the app container endpoint back.

The other details appear later.

Reply:
You're right, the router container is not something folks can interact with, at least atm. It's though important for them to understand this concept at some point, since they do see it as part of the architecture of their environment. 🤔  I'm comfortable leaving it as is and have this come up when the router is more visible